src/create-model.jl

```diff
@@ -663,6 +667,15 @@ function create_model(
                     ) for row in eachrow(df_flows)
                 ]
         @variable(model, 0 ≤ assets_investment[y in Y, a in Ai[y]])  #number of installed asset units [N]
+        @variable(
+            model,
+            0 <= assets_decommission_compact[
+                a in A, # we need a new set A, we don't want e.g., demand
+                y in Y,
+                v in V;
+                starting_year_compact[y, a] ≤ v < y, # double-check formulation; this is the domain, different from the sum
+            ]
+        )
```

Comment:
This variable should only exist for the assets that have the compact formulation. We need a new set with the assets that have the compact method